.github/workflows/notify-master-failure.yml

```diff
@@ -0,0 +1,25 @@
+name: 'Notify master failure'
+
+on:
+  workflow_run:
+    branches: 
+      - master
+    workflows:
+      - '**'
+    types:
+      - completed
+
+jobs:
+  slackNotify:
+    if: ${{ github.event.workflow_run.conclusion == 'failure' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Inform Slack users at Brim HQ of the new artifact
+        uses: tiloio/slack-webhook-action@v1.1.2
+        with:
+          slack_web_hook_url: ${{ secrets.SLACK_WEBHOOK_BRIMLABS_TEST }}
+          slack_json: |
+            {
+              "username": "github-actions",
+              "text": "Workflow \"${{ github.event.workflow_run.name }}\" failed on master.\n${{ github.event.workflow_run.html_url }}"
```

Comment:
Does `html_url` point to the failed run? The docs at https://docs.github.com/en/developers/webhooks-and-events/webhook-events-and-payloads#workflow_run don't seem to describe this field.